src/main/java/frc/robot/commands/AimAndFire.java

```diff
@@ -5,21 +5,20 @@
 import edu.wpi.first.wpilibj2.command.WaitCommand;
 import frc.robot.commands.Deprecated.MoveConveyor;
 import frc.robot.commands.Deprecated.ShooterOff;
-// import frc.robot.subsystems.ColorSensor;
 import frc.robot.subsystems.ConveyorSubsystem;
 import frc.robot.subsystems.Limelight;
 import frc.robot.subsystems.ShooterSubsystem;
 import frc.robot.subsystems.DrivetrainSubsystem;
-import frc.robot.subsystems.IntakeSubsystem;
 
 public class AimAndFire extends SequentialCommandGroup {
 
     public AimAndFire(ShooterSubsystem shooterSubsystem, ConveyorSubsystem conveyorSubsystem, Limelight limelight, DrivetrainSubsystem drivetrainSubsystem) {
         addCommands(
             //new AdjustConveyor(conveyorSubsystem, colorSensor),
-            new ParallelDeadlineGroup(new AimBotAngle(limelight, drivetrainSubsystem).withTimeout(0.7), new AimShotPID(shooterSubsystem, 3500)),
-            new AimShotCalculated(shooterSubsystem, limelight),
-            new WaitCommand(0.2),
+            new ParallelDeadlineGroup(new AimBotAngle(limelight, drivetrainSubsystem).withTimeout(0.7), new Shoot(shooterSubsystem, 3500)),
+            // new AimShotCalculated(shooterSubsystem, limelight),
+            new InstantCommand(() -> shooterSubsystem.setRPM(limelight.lerpRPM())),
```

Comment:
since setRPM is async and instant now, you could make a waitUntil statement until the RPM is at the desired speed

Reply:
although this works just as well, if we didn't reach our setpoint by 0.25 seconds, something has gone wrong haha